src/bokeh/models/annotations/geometry.py

```diff
@@ -249,6 +249,28 @@ def __init__(self, *args, **kwargs) -> None:
     The {prop} values for the line.
     """)
 
+    above_fill_props = Include(ScalarFillProps, prefix="above", help="""
+    The {prop} values for the area above the line.
+    """)
+
+    above_hatch_props = Include(ScalarHatchProps, prefix="above", help="""
+    The {prop} values for the area above the line.
+    """)
+
+    below_fill_props = Include(ScalarFillProps, prefix="below", help="""
+    The {prop} values for the area below the line.
+    """)
+
+    below_hatch_props = Include(ScalarHatchProps, prefix="below", help="""
+    The {prop} values for the area below the line.
```

Comment:
These could clarify what happens for inverted axes

Reply:
This would be better to explain in a common location for all those properties, possibly user guide.